docs/SSD diagrams/WebclientGetDeviceStatus.plantuml

```diff
@@ -0,0 +1,22 @@
+@startuml WebclientGetDeviceStatus
+
+entity IoTDevices as client
```

Comment:
move IoTdevice at the bottom of participants so it moves most left

Reply:
Updated.